bestpractices/querying_data_bestpractices.html.md.erb

```diff
@@ -4,6 +4,23 @@ title: Best Practices for Querying Data
 
 To obtain the best results when querying data in HAWQ, review the best practices described in this topic.
 
+## <a id="virtual_seg_performance"></a>Factors Impacting Query Performance
+
+The number of virtual segments used for a query directly impacts the query's performance. The following factors can impact the degree of parallelism of a query:
+
+-   **Cost of the query**. Small queries use fewer segments and larger queries use more segments. Some techniques used in defining resource queues can influence the number of both virtual segments and general resources allocated to queries. For more information, see [Best Practices for Using Resource Queues](managing_resources_bestpractices.html#topic_hvd_pls_wv).
+-   **Available resources at query time**. If more resources are available in the resource queue, those resources will be used.
+-   **Hash table and bucket number**. If the query involves only hash-distributed tables, the query's parallelism is fixed (equal to the hash table bucket number) under the following conditions: 
+  	- the bucket number (bucketnum) configured for all the hash tables is the same bucket number for all tables 
+   - the table size for random tables is no more than 1.5 times larger than the size allotted for the hash tables. 
```

Comment:
Seems like the MD formatting for this bullet is off - should line up with the previous nested bullet.  Also, it seems like this should read "1.5 times the size allotted" rather than "1.5 larger than the size allotted".